src/map/packets.h

```diff
@@ -2299,6 +2299,16 @@ packet(0x96e,-1,clif->ackmergeitems);
 	// New Packets End
 #endif
 
+#if PACKETVER >= 20130320
+// new packets
+// changed packet sizes
+	packet(0x09a7,10,clif->pBankDeposit,2,4,6); // CZ_REQ_BANKING_DEPOSIT
```

Comment:
If the 2013-03-20 client has these packets, then we should probably also update the condition to enable the banking system, since it still checks it against the 20130724 PACKETVER. See battle.c line 7592:

```
#if PACKETVER < 20130724
	if( battle_config.feature_banking ) {
		ShowWarning("conf/map/battle/feature.conf banking is enabled but it requires PACKETVER 2013-07-24 or newer, disabling...\n");
		battle_config.feature_banking = 0;
	}
#endif
```